content/04.baselines.md

```diff
@@ -1,14 +1,16 @@
+Depending on the amount and the nature of the available data, as well as the task to be performed, deep learning may not always be able to outperform conventional methods.
+As an illustration, Rajkomar et al. [@doi:10.1038/s41746-018-0029-1] found that simpler baseline models achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many. 
+Another example is provided by Koutsoukas et al., who benchmarked several traditional machine learning approaches against deep neural networks for modeling bioactivity data and found that, while well tuned deep learning approaches tend to outperform conventional classifiers on a moderately sized dataset, simple methods such as Naive Bayes classification tend to outperform deep learning as the noise in the dataset increases [@doi:10.1186/s13321-017-0226-y].
 
-It is worth noting that conventional machine learning methods (e.g., support vector machines, random forests) are also likely to benefit from parameter tuning.
-It can be tempting to train baseline models with these conventional methods using default parameters, which may provide acceptable but not stellar performance, but then tune the parameters for DL models to optimize performance.
-Hu and Greene [@doi:10.1101/385534] discuss a "Continental Breakfast Included" effect by which unequal hyperparameter tuning skews the evaluation of methods, especially those with performance that varies substantially with modest changes to hyperparameters.
-Those wishing to compare methods should tune the parameters of traditional and DL to optimize performance before making claims about relative performance differences.
-The performance comparison among DL models and many other ML approaches is informative only when the models are similarly tuned.
+It is worth noting that conventional off-the-shelf machine learning algorithms (e.g., support vector machines and random forests) are also likely to benefit from parameter tuning.
+It can be tempting to train baseline models with these conventional methods using default parameters, which may provide acceptable but not stellar performance, but then tune the settings for DL algorithms to further optimize performance.
+Hu and Greene [@doi:10.1101/385534] discuss a "Continental Breakfast Included" effect by which unequal hyperparameter tuning for different learning algorithms skews the evaluation of these methods, especially when the performance of an algorithm varies substantially with modest changes to its hyperparameters.
+Those wishing to compare of different learning algorithms should tune the parameters of traditional and DL-based methods for predictive performance optimization before making claims about relative performance differences.
```

Comment:
The phrasing here needs some work. The first part should instead be `Those wishing to compare different learning algorithms`. I also think the phrase `tune the parameters of traditional and DL-based methods for predictive performance optimization before` doesn't make any sense. Perhaps just changing it to `tune the parameters of both traditional and DL-based methods to optimize performance before`.

Reply:
oh yes! not sure what happened there -- that sounded terrible!